Documentation/ColumnsConfig/Type/Select/Tree/Index.rst

```diff
@@ -5,14 +5,35 @@
 selectTree
 ==========
 
-A field with the render type `selectTree` displays hierarchical data
-in a tree.
+
+The selectTree render type is used to display hierarchical data in a
+tree structure.
+
+A field with the selectTree render type allows you to represent data in
+a hierarchical manner, similar to a tree. It is typically used when working
+with database tables that have a hierarchical structure. The field requires
+certain fields to be set in order to function properly. The treeConfig and
+foreign_table fields are mandatory and must be provided to establish the
+connection to the relevant database table. Optionally, you can also use
+items or itemsProcFunc to pass the values, but these are not
+sufficient on their own. The top-level item in the tree will always represent
+the descriptive name of the table.
+
+Regarding joining several tables, the selectTree field can handle multiple
+tables through the configuration options. By appropriately setting up the
```

Comment:
The sentece is a little strange and does not really explain how to do that. Either explain how to do that or link to a source or just leave it out?